securedrop/db.py

```diff
@@ -158,7 +158,10 @@ class Reply(Base):
             order_by=id))
 
     source_id = Column(Integer, ForeignKey('sources.id'))
-    source = relationship("Source", backref=backref('replies', order_by=id))
+    source = relationship(
```

Comment:
I agree with your points here, and I definitely agree with you that the first is more readable than the second.  I’ll note that: 

```
    source = relationship("Source",
                          backref=backref("replies", order_by=id, cascade="delete"))
```

is not PEP8 compliant due to limits on characters per line. Instead I decided to opt for the ‘hanging intent’ style used in many of the tests in #1417 for readability. Happy to discuss further. 

Also: in order to address these kinds of style issues in the codebase, I suggest we tackle #47 and #886 soon and define and clean up the style in a few style-only PRs (I’m not going to change the lines you mentioned above in this PR since they’re not in the diff and I don’t want to confuse the history here, but the lines you point out are definitely something for a style PR once we’ve defined what constitutes good style for this project.)